packages/fast-components-react-base/src/listbox-item/listbox-item.props.ts

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+import {
+    ListboxItemClassNameContract,
+    ManagedClasses,
+} from "@microsoft/fast-components-class-name-contracts-base";
+
+export interface ListboxItemManagedClasses
+    extends ManagedClasses<ListboxItemClassNameContract> {}
+export interface ListboxItemUnhandledProps extends React.HTMLAttributes<HTMLDivElement> {}
```

Comment:
@nicholasrice to confirm, but I think because we are enumerating "id", we'll want to subtract that value from the HTMLAttribute interface we are extending here. We can use utility-types (existing dependency) to do this

Reply:
I think this is okay since it is the same type but we should validate